### ISSUE TITLE
Agent no longer supports runtime parameter "packages"?

### ISSUE THREAD

**Issue by tbee** [ebean-orm/ebean-agent#201]
The agent used to be able to be called with an packages argument like so:

```console
-javaagent:ebean-agent-4.9.2.jar=packages=com.my.package1,com.my.package2
```

It seems that in version 12.16.1 this argument is no longer supported? If so, this causes conflicts in a mixed environment where also JPA is present.

**Comment by rbygrave** [comment]
This was effectively replaced by https://github.com/ebean-orm/ebean-agent/issues/53

That is, we can control the packages that are included in enhancement by adding a `src/main/resources/ebean.mf` manifest file with entries for `entity-packages` (and `transactional-packages` if necessary).

**Comment by rbygrave** [comment]
> which ebean.mf is read? Probably just the one on the classpath, right?

Yes. It is loaded as a resource from the classpath.

**Comment by tbee** [comment]
This is not trivial in an EAR with all the module classloaders.

**Comment by tbee** [comment]
Since the java-agent is only used on development in combination with JRebel to hot reload, it would be nice to not have to wrestle with classloading in a JEE server, and be able to specify the file using something like a -Debean.mf=... argument.

**Comment by rbygrave** [comment]
I think it would be easier to just bring back support for `packages` such that we can do:
```
-javaagent:ebean-agent-4.9.2.jar=packages=com.my.package1,com.my.package2
```

**Comment by tbee** [comment]
Yes. That is what I've implemented in my patched version of ebean-agent. 

But it turns out the issue is less problematic than I initially feared: I've now got a single ebean.mf file, which is read by wildfly startup, and that also is copied into three JEE modules. That seems to fix my problem. But in theory it could be necessary to include the ebean.mf in many more JEE modules. 

Still waiting for the final tests by my colleagues.
